### ISSUE TITLE
sc2291 - tabs around -e in echo command; are they a problem?

### ISSUE THREAD

**Issue by pasha-19** [koalaman/shellcheck#2488]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2291
- My shellcheck version (`shellcheck --version` or "online"): 0.19.2 (from vscode)
- [x] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [x] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [ ] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash

echo    -e      'test'
echo    -e      "test"

```

#### Here's what shellcheck currently says:

```

$ shellcheck myscript
 
[Line 3:](javascript:setPosition(3, 1))
echo    -e      'test'
^-- [SC2291](https://github.com/koalaman/shellcheck/wiki/SC2291) (info): Quote repeated spaces to avoid them collapsing into one.
 
[Line 4:](javascript:setPosition(4, 1))
echo    -e      "test"
^-- [SC2291](https://github.com/koalaman/shellcheck/wiki/SC2291) (info): Quote repeated spaces to avoid them collapsing into one.

$
```

#### Here's what I wanted or expected to see:

I do not believe the tab '\t' before '-e' is an issue for sure.  The tab after '-e' could be debatable.  Since '-e' is not a potential operand to be printed, as echo uses it internally as a switch, I see no issue.  No  other command I have found yet objects to tabs between operands for readability.  Spaces needing protecting fall between operands to be concatenated for printing.  If one wants a leading space to be protected then it would need to be inside some form of quote and it will be rather obvious when testing, if needed.  In the above statement there is only a single printable and quoted non white space operand.

Single or double quotes change nothing.
